cves/CVE-2014-8547.yml

```diff
@@ -280,8 +277,8 @@ ipc:
     software system reads is another form of IPC.
 
     Answer should be boolean.
-  answer:
-  note:
+  answer: false
+  note: 'While the GIF is likely to be received over IPC, IPC is not responsible for the vulnerability'
```

Comment:
This makes sense. I was definitely overthinking the IPC process when filling out my YML. Your document has provided me with retrospective insight.